src/test/Button.test.js

```diff
@@ -0,0 +1,3 @@
+test('button clicks should be go back to the home page', () => {
+  expect(true).toBe(true);
+});
```

Comment:
- [ ] Dear @enocol, As mentioned by the previouse reviewer, `The tests you've provided seem to be unrelated to our Redux code's functionality.`, currently this test is taking `true` and expecting `true`. Having such a test will affirm you are not writing a valid test for the recruiters or anyone else who is reviewing your project. Please update your tests. `We are expected to test our application functionalities.` 

Please add a valid test for using `Jest` and `React testing library` to test whether your application is `rendering` components well, and `Redux` is operating as expected.